pages-activities/activities-learning.html

```diff
@@ -163,7 +163,21 @@ <h4>
             </div>
         </div>
     </div>
-
+    <div class="col-lg-4 col-md-6 d-flex align-items-stretch">
```

Comment:
Please move this block into alphabetical order with the rest of the items on the page.